### ISSUE TITLE
Crash if trying to create group story from settings

### ISSUE THREAD

**Issue by qwerty287** [signalapp/Signal-Android#12726]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Signal crashes if I try to create a new group story from settings.

### Steps to reproduce
- go to story settings
- new story
- try to create group story

**Actual result:** crash
**Expected result:** view my groups to create a new story

### Device info
**Android version:** 12 (Android One)
**Signal version:** 6.7.6

### Debug log

```
FATAL EXCEPTION: pool-18-thread-1
Process: org.thoughtcrime.securesms, PID: 7078
java.lang.NullPointerException
	at org.thoughtcrime.securesms.database.GroupTable$Reader.next(GroupTable.kt:1076)
	at org.thoughtcrime.securesms.database.GroupTable$Reader.next(GroupTable.kt:1023)
	at kotlin.sequences.FilteringSequence$iterator$1.calcNext(Sequences.kt:170)
	at kotlin.sequences.FilteringSequence$iterator$1.hasNext(Sequences.kt:194)
	at kotlin.sequences.SequencesKt___SequencesKt.count(_Sequences.kt:1236)
	at kotlin.sequences.SequencesKt.count(Unknown Source:0)
	at org.thoughtcrime.securesms.contacts.paged.collections.ContactSearchCollection.<init>(ContactSearchCollection.kt:20)
	at org.thoughtcrime.securesms.contacts.paged.ContactSearchPagedDataSource.createResultsCollection(ContactSearchPagedDataSource.kt:263)
	at org.thoughtcrime.securesms.contacts.paged.ContactSearchPagedDataSource.getCollectionSize(ContactSearchPagedDataSource.kt:101)
	at org.thoughtcrime.securesms.contacts.paged.ContactSearchPagedDataSource.getSectionSize(ContactSearchPagedDataSource.kt:89)
	at org.thoughtcrime.securesms.contacts.paged.ContactSearchPagedDataSource.size(ContactSearchPagedDataSource.kt:35)
	at org.signal.paging.BufferedPagingController.lambda$onDataInvalidated$1(BufferedPagingController.java:61)
	at org.signal.paging.BufferedPagingController.$r8$lambda$G8rug5SexbuP939rI5cW5vVT9HE(Unknown Source:0)
	at org.signal.paging.BufferedPagingController$$ExternalSyntheticLambda3.run(Unknown Source:2)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
	at java.lang.Thread.run(Thread.java:1012)
```

**Comment by alex-signal** [comment]
Hi, does this happen consistently for you? I am unable to reproduce.

**Comment by alex-signal** [comment]
@qwerty287 Thanks for the response! It looks like this should be fixed by commit 5b0084a5e2 which will be released in 6.8.x (due out in the middle of this week)
